### ISSUE TITLE
How to properly log/trace a missing lib?

### ISSUE THREAD

**Issue by Wooty-B** [ptitSeb/box86#644]
I am trying to run a program, rpcs3 for example, and I am getting the following error:

`Error loading needed lib libevdev.so.2`

The first thing I have done is make sure arm64 library is installed, then amd64 library. Even when copying one or the other into the running directory, I still get the error above.

I've tried running `BOX86_LOG=3 ./rpcs3` and `BOX86_TRACE=1 ./rpcs3`, however I'm not getting any different output or log files generated. Running an `strace` isn't giving me entirely what I need either, although I'm new to that. What is the proper way to troubleshoot this issue? Thanks!

**Comment by Jai-JAP** [comment]
You have to install armhf libraries for box86 wrapped ones and i386 ones for emulating them

**Comment by Wooty-B** [comment]
@Jai-JAP I had foolishly extracted just the shortcuts into their respective folders at `/lib/{arch}-linux-gnu`. Once I copied the linked AND actual file containing information, the app ran... Thanks for making me focus there!

I'm still looking for the former however, so I can have a clearer understanding of similar and other issues in the future. Not sure if I built with "Release" or "RelWithDebInfo" flags on compile, but will test that this evening.
